### DISCUSSION TITLE
Cannot see the bounding box in detect.py

### DISCUSSION THREAD

**Discussion by junzz827** [ultralytics/yolov5#2556] [Q&A]
Hello! Please help everyone.
I tried detct.py on raspberry Pi4 "aarch64" but I don't see the bounding box of Zidane & bus.jpg.
When I use 32bitOS with the same raspberry pi, it can be detected normally.
There is no error when using aarch64. The webcam also starts with source 0, but again the bounding box is not displayed.
The difference between the two operating systems is the version of Torch and Torch vision, do you think there is something to do with it?
aarch64:Torch 1.8.1   Torch vision 0.9.0 pycocotools 2.0.2
armv7l:Torch 1.7.0  Torch vision 0.8.0 pycocotools 2.0
Both Opencv are 4.5.1.48
The reason for using aarch64 is that pip yolov5 could not be installed properly on 32bit.

Thanking you in advance.

**Comment by glenn-jocher** [comment]
@junzz827 hopefully soon we'll have better official support for Raspberry Pi and Jetson Nano, in the meantime I see you are having to worry about pycocotools installs with detect.py, which I admit does not serve any purpose. One fix I can work on is creating an exclude list for dependency checks in detect.py. 

We created a PR for this purpose earlier in https://github.com/ultralytics/yolov5/pull/2041 that we can build on.

**Comment by glenn-jocher** [comment]
@junzz827 PR #2571 builds on #2041 to now exclude `pycocotools` and `thop` from detect.py dependency checks. 

This won't solve your problem completely, but at least now pycocotools won't be a factor and one less thing for you to worry about.

**Reply by junzz827** [reply]
Thank you for your reply, Mr. Glen. You will always be helpful.
I realized that pycocotools wasn't the cause and made some progress.
As a new initiative, I downgraded Pytorch and Torchvision to the same version as the successful 32bit OS, but it didn't help.
It seems that Pytorch's inference doesn't work well on raspberry pi 64bit OS for some reason. For the time being, we will move forward with 32bit. Thank you!
